class.jetpack-connection-banner.php

```diff
@@ -230,7 +230,7 @@ class="notice-dismiss"
 
 					<!-- slide 1: intro -->
 					<div class="jp-wpcom-connect__slide jp-wpcom-connect__slide-one jp__slide-is-active">
-						<h2><?php esc_html_e( 'Jetpack is here to enhance your site and your life', 'jetpack' ) ?></h2>
+						<h2><?php esc_html_e( 'Jetpack simplifies site security, customization, and promotion.', 'jetpack' ) ?></h2>
```

Comment:
@richardmuscat touchè! I'm just playing devil's advocate here. It's also worth mentioning that I pulled a lot of that text straight from the wordpress.org plugin repo. We should consider keeping them consistent. I'd love to see a lot of these additions you've made transfer over there!